src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -140,6 +140,13 @@ export class StemEligibilityView extends React.Component {
       <div className={divClassName}>
         <fieldset className="schemaform-field-template schemaform-first-field">
           <legend className={legendClassName}>
+            <div className="sr-only">
```

Comment:
While reviewing this code a second time on `localhost` it feels too repetitive to include the list of responses a second time. Let's just remove this block `<div className="sr-only">` and the list inside it. No additions will be necessary.

Reply:
Code removed